vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/GridPro.java

```diff
@@ -243,7 +243,7 @@ protected String getEditorType() {
      * @see EditColumnConfigurator#select(ItemUpdater, List)
      * @see #removeColumn(Column)
      */
```

Comment:
Could improve the javadocs to explicitly state that the renderer is used to display the content when the cell is not in edit-mode.

Reply:
Improved.